Manual de eventos sostenibles

CeliGaroe opened the pull request:
¡Hola! He creado un fork del spain-handbook para añadir[ la traducción del manual de eventos sostenibles](https://docs.google.com/document/d/1K28caiJrpn8pTXheMTV0w7NBwee5kJ7_tX6igghaOdE/edit) publicado en el [equipo de sostenibilidad global](https://make.wordpress.org/sustainability/handbook/sustainable-events/) ([enlace GitHub](https://github.com/WordPress/sustainability/tree/trunk/events)) aplicando las anotaciones que nos dejó @nilovelez en el documento de traducción. 

La idea es publicarlo [tal y como está en dicho equipo](https://make.wordpress.org/sustainability/handbook/sustainable-events/), con la página 'Manual de eventos sostenibles' como la madre, y el resto de secciones como subpáginas: 

![Captura de pantalla 2024-10-09 a las 10 54 08](https://github.com/user-attachments/assets/4350751f-3fc6-46a8-b32e-10e10d953045)

Una vez revisado y publicado, crearé un pull request secundario para añadir los enlaces internos correspondientes (entre subsecciones del manual). 

Si he hecho algo mal o necesitáis más información, soy toda oídos. :) ¡Gracias!

CeliGaroe committed:
Create manual-de-eventos-sostenibles.md

CeliGaroe committed:
Create ubicacion.md

CeliGaroe committed:
Create lugar-del-evento.md

CeliGaroe committed:
Create viajes.md

CeliGaroe committed:
Create patrocinadores.md

CeliGaroe committed:
Create catering.md

CeliGaroe committed:
Create regalos.md

CeliGaroe committed:
Create contenido.md

CeliGaroe committed:
Create marketing.md

CeliGaroe committed:
Create inspiracion.md

CeliGaroe committed:
Create consejos-asistentes.md

CeliGaroe committed:
Merge branch 'WordPress:main' into celigaroe-handbook-eventos-sostenibles-español

CeliGaroe committed:
Update regalos.md

jesusyesares commented on sostenibilidad/eventos/catering.md:
```diff
@@ -0,0 +1,13 @@
+- Revisa si el lugar donde se celebrará el evento tiene restricciones en la comida que puede traer la gente de catering y los patrocinadores.
+- Si el lugar del evento no tiene fuentes o sistemas para rellenar agua, habla con el catering o la empresa encargada de la comida.
+- A la hora de elegir una empresa de catering, busca una que cumpla con el mayor número de los siguientes criterios:
+  - Es un negocio local, que cocina comida de la zona, usa ingredientes locales y que idealmente elabora menús de temporada. También es recomendable    - que use productos de comercio justo, orgánicos o de origen vegetal, siempre y cuando sean locales.
```
Este guión no sé muy bien si va en un salto de página o si se ha escapado.

jesusyesares commented on sostenibilidad/eventos/regalos.md:
```diff
@@ -0,0 +1,21 @@
+# Regalos
+
+WordCamp Central te proporcionará [cintas para las acreditaciones y otros materiales de oficina](https://make.wordpress.org/community/handbook/wordcamp-organizer/planning-details/supplies/) que puedas necesitar para el evento.
```
Si no me equivoco, esto ya no es así. Confirmadlo.

jesusyesares reviewed (changes requested):
Hola @CeliGaroe, he marcado un par de cosillas, una pequeña (supuesta) errata y una cosa que hay que confirmar. Revisadla porfa.

Por lo demás, está para aprobar.

CeliGaroe committed:
Update catering.md

Eliminado el guión extra que no pintaba nada ahí. ¡Gracias, Jesús!

CeliGaroe committed:
Update regalos.md

Cambiado el comienzo para evitar la mención a los envíos que hacía antes WordPress Central, ya que están parados desde la pandemia.

jesusyesares reviewed (dismissed):
Todo correcto :)

CeliGaroe committed:
Update index.md

Actualización de index.md para crear la página https://es.wordpress.org/make/handbook/sostenibilidad/

javiercasares committed:
Merge branch 'main' into celigaroe-handbook-eventos-sostenibles-español

javiercasares committed:
reorganización de estructura

javiercasares committed:
Merge branch 'celigaroe-handbook-eventos-sostenibles-español' of https://github.com/CeliGaroe/spain-handbook into pr/356

javiercasares committed:
revisión, corrección y formato.

javiercasares committed:
Merge branch 'main' into celigaroe-handbook-eventos-sostenibles-español

CeliGaroe committed:
Update index.md

CeliGaroe committed:
Update index.md

CeliGaroe committed:
Update index.md

CeliGaroe committed:
Update index.md

CeliGaroe committed:
Update index.md

CeliGaroe committed:
Update index.md

CeliGaroe committed:
Update index.md

javiercasares committed:
Merge branch 'main' into celigaroe-handbook-eventos-sostenibles-español

javiercasares committed:
Nueva ruta para los eventos

javiercasares committed:
versión correcta.

javiercasares committed:
Merge branch 'main' into celigaroe-handbook-eventos-sostenibles-español

javiercasares committed:
Merge branch 'main' into celigaroe-handbook-eventos-sostenibles-español

javiercasares committed:
Merge branch 'main' into celigaroe-handbook-eventos-sostenibles-español

javiercasares committed:
manifest y ajustes

javiercasares committed:
fix
